request_legacy.go

```diff
@@ -0,0 +1,18 @@
+// +build !go1.13
```

Comment:
It would be a good idea to have a comment here and link what has changed in go v1.13 and why it is necessary.

Reply:
I totally agree with you. It is important for future maintainers. I will do it.